src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -28,15 +31,16 @@
    * ".rscc" is a hidden folder in the user's home directory (e.g. /home/user)
    */
   private static final String RSCC_FOLDER_NAME = ".rscc";
+  private static final String STUN_DUMP_FILE_NAME = "ice4jDemoDump.ice";
```

Comment:
I think this is not needed, should be refactored, as soon as the dump is generated dynamically. Maybe you should add a comment